internal/fs/tarfs/tarfs.go

```diff
@@ -108,11 +108,11 @@ func (tfs *TarFS) Stat(name string) (fs.FileInfo, error) {
 // getEntry returns the named entry.
 func (tfs *TarFS) getEntry(name string) (*entry, error) {
 	if !fs.ValidPath(name) {
-		return nil, &fs.PathError{Path: name, Err: fs.ErrInvalid}
+		return nil, &fs.PathError{Op: "open", Path: name, Err: fs.ErrInvalid}
 	}
 	entry, ok := tfs.entries[name]
 	if !ok {
-		return nil, &fs.PathError{Path: name, Err: fs.ErrNotExist}
+		return nil, &fs.PathError{Op: "open", Path: name, Err: fs.ErrNotExist}
```

Comment:
Looks like we should define a constant for "open".

Reply:
I don't know, just check `os` package and seems they doesn't add a const
https://cs.opensource.google/go/go/+/refs/tags/go1.22.5:src/os/file_windows.go

Reply:
Nevermind, the comment became invalid now.